client/css/style.css

```diff
@@ -1038,11 +1038,11 @@ kbd {
 }
 
 #chat .self .text {
-	color: #999;
+	color: #444;
 }
 
 #chat .msg.channel_list_loading .text {
-	color: #999;
+	color: #444;
```

Comment:
That's fine, I didn't even see that one was bright, good catch.

Turns out, `#chat table.channel-list td` (the content of the table when loaded) is `#555` at the moment. You could make it `#444` to match your change here, or even better IMO, remove the color definition here and in `#chat table.channel-list td` so they both match "normal" color, i.e. `#222`.
What do you think?